applications/asset_tracker_v2/tests/gps_module/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(gps_module_test)
+
+# generate runner for the test
+test_runner_generate(src/gps_module_test.c)
+
+# create mock
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/event_manager.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/date_time.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/applications/asset_tracker_v2/src/modules/modules_common.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/drivers/gps.h drivers )
+cmock_handle(${ZEPHYR_BASE}/include/sys/reboot.h sys)
+
+
+# add gps_module (the unit under test)
+target_sources(app PRIVATE ../../src/modules/gps_module.c)
+
+# add test file
+target_sources(app PRIVATE src/gps_module_test.c)
+
+target_include_directories(app PRIVATE .)
+target_include_directories(app PRIVATE ../../src/)
+target_include_directories(app PRIVATE ../../src/modules/)
+target_include_directories(app PRIVATE ../../../../subsys/event_manager)
```

Comment:
Line 32 and 33 should be kept as it was I think. This will not work for the out-of-tree scenario. In that case the Asset Tracker v2 application directory is at the same level as the sdk-nrf directory.

**asset_tracker_v2** **nrf** **zephyr** ...

Reply:
```suggestion
target_include_directories(app PRIVATE ${ZEPHYR_BASE}/../nrf/subsys/event_manager)
```